application/borrower/model.py

```diff
@@ -43,6 +44,16 @@ def get_by_id(id_):
     def get_by_token(token_):
         return Borrower.query.filter_by(token=token_).first()
 
+    def get_by_verify_pid(verify_pid):
+        return Borrower.query.join(VerifyMatch).filter(VerifyMatch.verify_pid == verify_pid).first()
+
+
+class VerifyMatch(db.Model):
+    __tablename__ = 'verify_match'
+
+    verify_pid = db.Column(db.String, primary_key=True)
+    borrower_id = db.Column(db.Integer, ForeignKey("borrower.id"), primary_key=True)
+
 
 def bin_to_char(bin_str):
```

Comment:
We might have to merge master back into this.  I am guessing that the 6 magic number in this function is the length of the token; which might have changed to 8 ?